positions/views.py

```diff
@@ -0,0 +1,41 @@
+from django.shortcuts import render
+from django.views import generic
+from django.urls import reverse
+import datetime
+
+from .models import Position
+from .forms import UpdateCreatePosition
+
+from django.contrib.auth.mixins import PermissionRequiredMixin, LoginRequiredMixin
+
+# Create your views here.
+
+class CreatePosition(PermissionRequiredMixin, LoginRequiredMixin, generic.CreateView):
+    model = Position
+    form_class = UpdateCreatePosition
+    template_name="form_crispy_cbv.html"
+    permission_required="positions.add_position"
+    
+    def get_success_url(self):
+        return reverse("accounts:me")
```

Comment:
This seemed a bit odd in my opinion and it definitely caught me by surprise. I think the user would be expecting to end up on either the position detail page or the open positions list.